pomegranate/base.pyx

```diff
@@ -137,6 +138,29 @@ cdef class Model(object):
 		"""
 		raise NotImplementedError
 
+	def plot( self, n=1000, **kwargs ):
```

Comment:
I can't think of a better implementation right now, but that method has some issues. In particular it assumes a diagonal covariance matrix which can be misleading. I suppose it's better than not having anything at all right now, as long as it's properly documented to show this limitation.